src/ModelMerger.py

```diff
@@ -0,0 +1,158 @@
+import cobra
+import pandas as pd
+import re
+from src.MeMoModel import MeMoModel
+#from cobra.core import DictList
+
+
+
+class ModelMerger:
+
+    def __init__(self,
+                 memo_model: MeMoModel = None,
+                 matches: pd.DataFrame = None) -> None:
+        self.memo_model = memo_model
+        self.matches = matches.copy()
+        self.matches = self.matches.rename(columns={'met_id2': 'source_namespace'})
+        self.matches = self.matches.rename(columns={'met_id1': 'target_namespace'})
+
+    def convert_exchange_rxns_to_translation_rxns(self):
+        """converts exchange reactions to namespace translation reactions and adds their respective compartment"""
+        cobra_model = self.memo_model.cobra_model
+        for ex in cobra_model.exchanges:
+            if re.match(r"^EX_", ex.id):
+                # Replace "EX_" with "TR_" at the start of the string
+                new_id = re.sub(r"^EX_", "TR_", ex.id)
+                # Update the reaction ID
+                ex.id = new_id
+                if len(ex.metabolites.keys()) == 1:
+                    met = list(ex.metabolites.keys())[0]
+                    # create the metabolite in the translation compartment and give it a +1 stoichiometry (production)
+                    met_t = met.copy()
+                    #  TODO: replace regular expression with function call for the remove suffix function
+                    #   (need to modify the handle_metabolites_prefix_suffix_function)
+                    met_t.id = re.sub(r"[^_]+$", "t", met.id)  # substitutes the compartment suffix (all
+                                                                            # that follows the last underscore) with
+                                                                            # the new compartment's symbol
+                    met_t.compartment = "t"
```

Comment:
@SteFlor verify if line 37 automatically creates the new compartment, and describe this in a comment